phantomjs/bridge.js

```diff
@@ -9,7 +9,7 @@
 /*global QUnit:true, alert:true*/
 (function (factory) {
   if (typeof define === 'function' && define.amd) {
-    require(['qunit'], factory);
+    define('phantom-bridge', ['qunit'], factory);
```

Comment:
@Arkni , this recommendation isn't quite apply to this case - as script is injected and there is no path info. Also module name contains no path. And after all, it isn't production code but merely a set of tests.
My call for this PR is usability. These links you've provide require tests bootstrapped in a very specific way and if something isn't right one has to spend some time to investigate, understand the problem and google for solution.
The alternative - have named module, a line in documentation saying you have to require it if you're using requirejs and flexibility to design application in any way plus absence of frustration and having everything working 'out of the box'.